src/test/kotlin/javabot/operations/URLTitleOperationTest.kt

```diff
@@ -15,15 +15,22 @@ import org.testng.annotations.Test
 class URLTitleOperationTest : BaseTest() {
     @Inject
     private lateinit var operation: URLTitleOperation
+
     @Inject
     private lateinit var config: JavabotConfig
     private val analyzer = URLContentAnalyzer()
 
     @Test(dataProvider = "urls")
     fun testSimpleUrl(url: String, content: String?) {
         val results = operation.handleChannelMessage(Message(TEST_CHANNEL, TEST_USER, url))
+        var testContent = true
         if (content != null) {
-            if (url.contains("twitter") && config.twitterAccessToken() != "") {
+            if (url.contains("twitter")) {
+                if (config.twitterAccessToken() == "") {
```

Comment:
do we actually use any of the twitter token stuff anymore?  don't we just parse headers/html?

Reply:
and yes, the twitter stuff doesn't work without the token, for whatever reason